company/okrs/2020_q1.md

```diff
@@ -0,0 +1,105 @@
+
+## Product
+
+1. **Product: Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Each feature is reviewed before release to ensure quality UX.
+1. **Product: Reduce product friction to accelerate customers through pipeline**
+   1. Product KR: Shorten onboarding time of "intent to try Sourcegraph" to "whole organization is using Sourcegraph".
+   1. Product KR: Shorten product feedback cycle on new features.
+1. **Product: Build the product and design team to support engineering and ARR growth**
+   1. Product KR: Grow team to plan (+1 UX Designer, +1 PM).
+   1. Product KR: Handbook describes product strategy and processes.
+   1. Product KR: Customer feature prioritization process scales with team and customer growth.
+
+Build out vision for product and run customer validation discussions
+
+## Engineering
+
+1. **VP Eng: Build a world class engineering team.**
+   1. VP Eng KR: Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. VP Eng KR: Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. VP Eng KR: Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **VP Eng: Create a data driven planning process.**
+   1. VP Eng KR: Each item in a tracking issue has an estimate of time or size.
+   1. VP Eng KR: Each team commits to a written work plan for each release cycle that accounts for the work estimates and the team's capacity.
+   1. VP Eng KR: Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+1. **Distribution: Resolve issues for critical customer deals and expansions**
+1. **Distribution: Improve developer productivity**
+   1. Distribution KR: Lower build times to X
+   1. Distribution KR: Automate away release captain role
+   1. Distribution KR: Infrastructure for automated deploys and releases (no human in loop required by default)
+1. **Distribution: Build features that accelerate customers through pipeline**
+   1. Distribution KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
+   1. Distribution KR: Ensure new features (LSIF, GitLab Native Integration, Automation) are accessible and easily tryable by customers once they go GA
+   1. Distribution KR: Surface information to site admins and managers to encourage expansion/upsell
+
+1. **Code intelligence: LSIF is adopted by customers**
+   1. Code intelligence KR: At least three customers are using LSIF on their primary repositories
+   1. Code intelligence KR: At least one customer is using LSIF across all of their code in languages for which there exists an LSIF indexer
+   1. Code intelligence KR: Start tracking the performance (user interaction to result) of hover tooltips, go to definition, and find references
```

Comment:
I believe there's urgency to fixing our end to end user perceived latency so I'd like to suggest we aim for more than just start measuring it this quarter (as we're doing for search in core services).

```suggestion
   1. Code intelligence KR: 99th percentile of end to end latency of hover tooltips, go to definition, and find references is below N seconds.
```

Framing it like this ensure we focus on user perceived latency and it may require you to split this KR into multiple ones due to the potentially differing complexities of making each of those user interactions faster. We'll probably need things like https://github.com/sourcegraph/sourcegraph/issues/1423 to measure it.